skills/aiq-research/skill-card.md

```
@@ -26,16 +26,15 @@ Mitigation: Review and scan skill before deployment. <br>
 
 ## Reference(s): <br>
 - [NVIDIA AI-Q Blueprint Repository](https://github.com/NVIDIA-AI-Blueprints/aiq) <br>
-- [DeepResearch Bench Paper](https://arxiv.org/pdf/2506.11763) <br>
-- [DeepResearch Bench Leaderboard](https://huggingface.co/spaces/muset-ai/DeepResearch-Bench-Leaderboard) <br>
 - [Helper script](scripts/aiq.py) <br>
+- [Deployment and backend validation](../aiq-deploy/SKILL.md) <br>
 
 
 ## Skill Output: <br>
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)